sdcm/cluster.py

```diff
@@ -5879,6 +5879,24 @@ def save_sct_dashboards_config(self, node):
         node.remoter.run('mkdir -p {}'.format(sct_monitoring_addons_dir), ignore_status=True)
         node.remoter.send_files(src=self.sct_dashboard_json_file, dst=sct_monitoring_addons_dir)
 
+    def update_default_time_range(self, start_timestamp: float, end_timestamp: float) -> None:
```

Comment:
let add a call to specific it on start of the monitoring stack

Reply:
Isn't it already there? `tester.py:2975`

Reply:
yes, but in case of CI test timeout we might not get to 'teardown'. So idea is to update also at the beginning of a test and later update to proper values.